Makefile

```diff
@@ -1,7 +1,7 @@
 GO ?= go
 GORELEASER ?= goreleaser
 
-OTELCOL_BUILDER_VERSION ?= 0.63.1
+gitOTELCOL_BUILDER_VERSION ?= 0.64.0
```

Comment:
```suggestion
OTELCOL_BUILDER_VERSION ?= 0.64.0
```

Reply:
Thanks for fixing my typo @TylerHelmuth!